### PR TITLE
Fixes #2: adding source location support

### LINKED ISSUE #2: Record and output LOC

### PR DESCRIPTION
**Pull request by ikarienator**
@michaelficarra I honestly don't know how to test it.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -19,3 +19,9 @@ import {Parser} from "./parser";
 export default function parse(code) {
   return new Parser(code).parseScript();
 }
+
+export function parseWithLocation(code) {
```
Comment (michaelficarra) [inline src/index.js]: This should probably not be a separate interface, and instead be an optional second parameter to `parse`.

**Comment by michaelficarra** [comment]
Can you rebase this and address my comment?

**Comment by ikarienator** [comment]
Shall we discuss about the design of the interface?

**Comment by michaelficarra** [comment]
Sure, let's talk in the morning.

**Comment by ikarienator** [comment]
@michaelficarra We shall merge this.

**Pull request merged**